lib/document.js

```diff
@@ -2223,8 +2223,9 @@ Document.prototype[documentModifiedPaths] = Document.prototype.modifiedPaths;
  * @api public
  */
 
-Document.prototype.isModified = function(paths, modifiedPaths) {
+Document.prototype.isModified = function(paths, options, modifiedPaths) {
```

Comment:
the jsdoc needs to be updated to reflect the new parameter, also is `options` optional or required?

Reply:
Optional. I updated jsdoc in 0960fae